packages/utils/src/crypto/packet/constants.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Hash algorithm that is used to derive the shared secrets
+ * and its output length
+ */
+export const HASH_ALGORITHM = 'blake2s256'
```

Comment:
This is a duplicate of packages/core/src/messages/constants.ts

Reply:
This is intentional because the packet creation and transformation logic is supposed to be independent from PoR and especially packet acknowledgement and can use different crypto primitives such as different hash lengths